.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        pg-version: ['9.6', '10', '11']
```

Comment:
how come we're still doing 9.6 here?

Reply:
Can remove that, I had assumed we still want to support that for the time being